csharp/extractor/Semmle.Extraction.CSharp.Standalone/BuildAnalysis.cs

```diff
@@ -434,12 +486,16 @@ private void AnalyseSolutions(IEnumerable<string> solutions)
             });
         }
 
-        public void Dispose()
-        {
-            packageDirectory?.Dispose();
-        }
+        public void Dispose() => packageDirectory?.Dispose();
 
-        [GeneratedRegex("<PackageReference .*Include=\"(.*?)\".*/>", RegexOptions.IgnoreCase | RegexOptions.Compiled | RegexOptions.Singleline)]
+        [GeneratedRegex("<PackageReference\\s+Include=\"(.*?)\".*/?>", RegexOptions.IgnoreCase | RegexOptions.Compiled | RegexOptions.Singleline)]
```

Comment:
I think this change might break cases when the package version is specified before the `Include=`.